sympy/simplify/combsimp.py

```diff
@@ -217,6 +219,232 @@ def explicate(p):
                 denom_gammas.extend(l)
             elif isg is False:
                 denom_others.extend(l)
+        # =========== level >= 2 work: factor absorbtion =========
+        if level >= 2:
```

Comment:
@jksuom I looked into "find_fuzzy" function and the definitions of all functions involved in it. But, I am not sure how to proceed to make it deal with pure combinatorial functions. I am not quite familiar with the functions in other modules, so please help me. Thanks